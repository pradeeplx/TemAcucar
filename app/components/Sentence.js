import React from 'react'
import { Text, Platform } from 'react-native'
import { fontFactor } from "../helpers"
import Colors from "../Colors"

export default Sentence = (props) => (
  <Text style={[{
    color: Colors.brown,
    fontFamily: (Platform.OS === 'ios' ? 'Avenir' : 'Roboto'),
    fontSize: 12 * fontFactor(),
  }, props.style]}>
    {props.children}
  </Text>
)
