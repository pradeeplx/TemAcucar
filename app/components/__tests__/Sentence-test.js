import 'react-native'
import React from 'react'
import Sentence from '../Sentence'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Sentence style={{top: 0}}>FooBar</Sentence>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
