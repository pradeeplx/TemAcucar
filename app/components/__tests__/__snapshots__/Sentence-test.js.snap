exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Object {
        "top": 0,
      },
    ]
  }>
  FooBar
</Text>
`;
